bundles/org.openhab.binding.daikin/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -102,26 +109,14 @@
 		<label>Set Point</label>
 		<description>The set point temperature</description>
 		<category>Temperature</category>
+		<tags>
+			<tag>Setpoint</tag>
+			<tag>Temperature</tag>
+		</tags>
 		<state pattern="%.1f %unit%" step="0.5"/>
 		<autoUpdatePolicy>veto</autoUpdatePolicy>
 	</channel-type>
 
-	<channel-type id="acunit-indoortemp">
-		<item-type>Number:Temperature</item-type>
-		<label>Indoor Temperature</label>
-		<description>The indoor temperature</description>
-		<category>Temperature</category>
-		<state readOnly="true" pattern="%.1f %unit%" step="0.5"/>
```

Comment:
Do you remember if default step is 1? In that case I guess replacing by system channel type could have UI impact?

Reply:
Would `step` affect a read-only channel?